nx/test/nx_test.exs

```diff
@@ -1628,4 +1628,69 @@ defmodule NxTest do
                )
     end
   end
+
+  if Version.match?(System.version(), ">= 1.13.0-dev") do
+    describe "sigil" do
+      test "evaluates to tensor" do
+        import Nx
+
+        assert ~M[-1 2 3 4] == Nx.tensor([-1, 2, 3, 4])
+        assert ~M[1
+                  2
+                  3
+                  4] == Nx.tensor([[1], [2], [3], [4]])
+        assert ~M[1.0 2  3
+                  11  12 13] == Nx.tensor([[1.0, 2, 3], [11, 12, 13]])
```

Comment:
Please don't forget to format this file. :)

Reply:
That's how it looks after running `mix formatter`, I double-checked :)

Reply:
Interesting. It may be a formatter bug. :)